uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -32,8 +32,11 @@ define([
       this.absencePeriods = [];
       this.absenceTypes = [];
       this.calendar = {};
-      this.contact = {};
+      //1. the contact is selected by manager when applying absence on behalf of manager
+      //2. the contact is set when manager is responding to request to display name
+      this.contact = null;
```

Comment:
Why is this even needed? Can't you use the `contact_id` property of the leave request instance?